client/src/components/organisms/Navbar/Navbar.jsx

```diff
@@ -62,12 +58,122 @@ function Navbar() {
             <div className="nav-CTA">
                 {(isUserLoggedIn)?(<Button kind="filled" onClick={handleLogout}>Logout</Button>):
```

Comment:
You can add a handler here. The user may accidentally click on logout. 
Suggestion:  Ask the user of he is sure to logout, and then delete from localStorage